server/src/main/java/org/apache/iotdb/db/metadata/MManager.java

```diff
@@ -1121,7 +970,6 @@ public StorageGroupMNode getStorageGroupNodeByPath(PartialPath path) throws Meta
    */
   public MNode getDeviceNodeWithAutoCreateAndReadLock(
       PartialPath path, boolean autoCreateSchema, int sgLevel) throws MetadataException {
```

Comment:
If there is no read lock anymore, what about change the method name to `getDeviceNodeWithAutoCreate`?

Reply:
fixed